### PR TITLE
fix http link on home

### PR DESCRIPTION
**Pull request by elgatoflaco**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5863,6 +5867,15 @@ react-error-overlay@^4.0.0:
   version "4.0.0"
   resolved "https://registry.yarnpkg.com/react-error-overlay/-/react-error-overlay-4.0.0.tgz#d198408a85b4070937a98667f500c832f86bd5d4"
 
+react-helmet@^5.2.0:
```
Comment (Varunram) [inline yarn.lock]: why is this thing here? Ideally, it shouldn't be since I merged the other PR (`yarn lock` is a pain)

**Comment by Varunram** [comment]
@elgatoflaco?

File: yarn.lock (hunk 1 above)
Comment (elgatoflaco) [reply yarn.lock]: I'm sorry, new to github. What can I do to solve it?

**Comment by Varunram** [comment]
Merged via https://github.com/Musicoin/website/commit/9ad5a82395492111c829c0983fa383c8a6461ee6

File: yarn.lock (hunk 1 above)
Comment (Varunram) [reply yarn.lock]: Ah, never mind, I did it myself

**Comment by Varunram** [comment]
You should pull from master now and reset all your local changes (SEO changes, etc).

File: yarn.lock (hunk 1 above)
Comment (elgatoflaco) [reply yarn.lock]: mercí beaucoup :)

**Pull request closed without merging**